index.html

```diff
@@ -41,7 +41,7 @@
                 </a>
                 <div id="testContainer"></div>
                 <div id="progress">
-                    <div id="progress-completed"></div>
+                    <progress id="progress-completed"></progress>
```

Comment:
I think this should have an `aria-label` or be wrapped inside a `<label>` element as per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#labelling

Reply:
Added `aria-label`, thanks for the review.